src/OpenLoco/Vehicles/VehicleManager.cpp

```diff
@@ -22,25 +22,38 @@ namespace OpenLoco::VehicleManager
     }
 
     // 0x004279CC
-    static void vehiclePickupWater(EntityId head, uint8_t flags)
+    void vehiclePickupWater(EntityId head, uint8_t flags)
     {
         registers regs;
         regs.di = enumValue(head);
         regs.bl = flags;
         call(0x004279CC, regs);
     }
 
-    // 0x004AF0B2
-    static void vehiclePickupAir(EntityId head, uint8_t flags)
+    // 0x00426B29
+    void vehiclePickupAir(EntityId head, uint8_t flags)
     {
         registers regs;
         regs.di = enumValue(head);
         regs.bl = flags;
-        call(0x004AF0B2, regs);
+        call(0x00426B29, regs);
```

Comment:
Ouch, does this mean the bug was in the game before this PR? That's worth a changelog entry then.

Reply:
Well it's in a function that i don't think ever really happens so it's not like it will get called much